File: MyMQTT2/app/src/main/java/com/example/mymqtt/MyApplication.java
package com.example.mymqtt;

import android.app.Application;

import com.example.mymqtt.sharedpreference.DataLocalManager;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        DataLocalManager.init(getApplicationContext());
    }
}

File: BLE/app/src/main/java/com/example/ble/MyApplication.kt
package com.example.ble

import android.app.Application
import android.graphics.Region
import com.estimote.coresdk.observation.region.mirror.MirrorRegion
import com.estimote.coresdk.service.BeaconManager

class MyApplication : Application() {

//    private lateinit var beaconManager:BeaconManager

    override fun onCreate() {
        super.onCreate()
//        beaconManager = BeaconManager(applicationContext)

    }
}